contents/docs/self-host/deploy/troubleshooting.md

```diff
@@ -42,56 +20,8 @@ nslookup <your-hostname> 1.1.1.1
 
 Note that when using a browser there are various layers of caching and other logic that could make the resolution work (temporarily) even if its not correctly set up.
 
-### Kafka crash looping (disk full)
-
-You might see an error similar to this one in the Kafka pod:
-
-```
-Error while writing to checkpoint file /bitnami/kafka/data/...
-java.io.IOException: No space left on device
-```
-
-This tells us that the data disk is full. To resize the disk, please follow the [runbook](/docs/runbook/services/kafka/resize-disk).
-
-#### Why did we run into this problem and how to avoid it in the future?
-
-There isn't a way for us to say "if there's less than X% of disk space left, then nuke the oldest data". Instead we have two conditions that restrict, when stuff can be deleted:
-
--   size (`logRetentionBytes: _22_000_000_000`) for the minimum size of data on disk before allowed deletion.
--   time (`logRetentionHours: 24`) for the minimum age before allowed deletion.
-
-We need to configure these well, but a disk monitoring utility can help catch this problem before we end up in a crash loop.
-
-See more in these stack overflow questions ([1](https://stackoverflow.com/questions/52970153/kafka-how-to-avoid-running-out-of-disk-storage), [2](https://stackoverflow.com/questions/53039752/kafka-how-to-calculate-the-value-of-log-retention-byte), [3](https://stackoverflow.com/questions/51823569/kafka-retention-policies)).
-
-### Upgrade failed due to cert-manager conflicts
-
-If a deploy fails with the following error:
-
-```
-Error: UPGRADE FAILED: rendered manifests contain a resource that already exists. Unable to continue with update: CustomResourceDefinition "certificaterequests.cert-manager.io" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "posthog"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "posthog"
-```
-
-The issue might be with cert-manager custom resource definitions already existing and being unupgradable.
-
-Try running helm upgrade without `--atomic` to fix this issue.
-
-### Namespace deletion stuck at `terminating`
-
-While deleting the namespace, if your Helm release uses `clickhouse.enabled: true` you might end up in the operation being indefinitely stuck.
-
-This is a [known behavior](https://github.com/Altinity/clickhouse-operator/issues/1043) of the `clickhouse-operator` finalizer. Workaround:
-
--   patch CHI removing the finalizer: `kubectl patch chi posthog -n posthog -p '{"metadata":{"finalizers":null}}' --type=merge`
-
--   delete CHI: `kubectl delete chi posthog -n posthog`
-
 ## FAQ
 
-### How can I increase storage size?
-
-To increase the storage size of the ClickHouse, Kafka or PostgreSQL service, take a look at our [runbook](/docs/runbook) section.
```

Comment:
also remove 
1. `### How do I see logs for a pod?`
2. ### How do I connect to Postgres?

pod is a k8s concept (you can see `kubectl` there -> k8s not something one would use on docker compose